spec/helpers/firm_helper_spec.rb

```diff
@@ -54,4 +54,31 @@
       it { is_expected.to eq('2.0 miles away') }
     end
   end
+
+  describe 'minimum_pot_size_text' do
+    let(:available_ordinals) { I18n.t('investment_size.ordinal').keys.map(&:to_s).map(&:to_i) }
```

Comment:
Might it be a good idea to add a comment here to explain the external information that this class relies on? i.e. why are you getting ordinals from the translations, what happens if the translations change in future (will it be obvious why this test has started failing) etc.